database/migrations/2024_07_05_140225_alter_site_polygon_table_to_add_versioning_attributes.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('site_polygon', function (Blueprint $table) {
+            $table->string('version_name')->nullable();
+            $table->string('primary_uuid')->nullable()->after('uuid');
+            $table->boolean('is_active')->default(false);
```

Comment:
This one is not a boolean and references has been updated to true / false accordingly.